docs/technical-support.md

```diff
@@ -1,26 +0,0 @@
----
```

Comment:
was this a dupe?

Reply:
poked around git a bit and best guess is `technical-support` was originally here and later moved under the `faq` section. the doc was left hanging when that happened.